src/pages/Flow/Settings/Schedules.vue

```diff
@@ -290,10 +321,22 @@ export default {
                   />
                 </div>
                 <div v-show="selectedTab === 1">
+                  <p
+                    v-if="checkDefualtParameters(allDefaultParameters)"
+                    class="mt-8 text-body-1"
+                    >If a clock has default parameters then that will override
+                    the flow group and flow parameters. If not, the flow group
+                    paramter will be used, else it will use the flow parameter.
```

Comment:
```suggestion
                    parameter will be used, else it will use the flow parameter.
```

Reply:
Probably should not just be ", else". Perhaps it should just say something like

Parameters will be resolved in the following order: clock values, flow group values, flow values. If a parameter value is set in an earlier item, it will not be loaded from a later one.

Also shouldn't flow settings have precedence over flow group settings? i.e. clock > flow > flow group

Reply:
Thanks - I've included some of that wording in my update.